src/overview.stories.mdx

```diff
@@ -0,0 +1,483 @@
+
+<Meta title="Overview" />
+
+export const StickerContainer = styled.div`
+  background: ${getColor('neutral.300')};
+  border-radius: ${getRadii('default')};
+  padding: ${getSpace('md')};
+`;
+
+export const StickerModalItem = () => {
+  const [isShown, setIsShown] = useState();
+  const handleButtonClick = (e) => {
+    e.preventDefault();
+    setIsShown(!isShown);
+  };
+  return (
+    <>
+      <Button onClick={handleButtonClick}>Open modal</Button>
+      {isShown && (
+        <SemanticModal
+          actions={[
+            {
+              label: 'Secondary',
+              name: 'Secondary',
+              onClick: noop,
+            },
+            {
+              label: 'Primary',
+              name: 'Primary',
+              onClick: noop,
+            },
+          ]}
+          onClose={() => setIsShown(false)}
+          message="You can replace this with a local component. You can also just override the text, but keep in mind that you might lose content that way."
+          title="This is a placeholder"
+          primaryButtonColor={ButtonColors.primary}
+        />
+      )}
+    </>
+  );
+};
+
+export const StickerDrawerItem = () => {
+  const [isVisible, setVisibility] = useState(false);
+  const handleButtonClick = (e) => {
+    e.preventDefault();
+    setVisibility(!isVisible);
+  };
+  return (
+    <>
+      <Button onClick={handleButtonClick}>Open drawer</Button>
+      {isVisible && (
+        <Drawer
+          size="sm"
+          footer={<div>Footer</div>}
+          title="Drawer"
+          onClose={() => setVisibility(false)}
+        >
+          <div>Content</div>
+        </Drawer>
+      )}
+    </>
+  );
+};
+
+export const StickerToastItem = () => {
+  const [isVisible, setVisibility] = useState(false);
+  const handleButtonClick = (e) => {
+    e.preventDefault();
+    setVisibility(!isVisible);
+  };
+  return (
+    <>
+      <Button onClick={handleButtonClick}>Toggle toast</Button>
+      {isVisible && (
+        <Toast
+          onClose={() => {
+            setVisibility(false);
+          }}
+          variant="info"
+        >
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Morbi non
+            massa a lacus posuere.
+          </p>
+        </Toast>
+      )}
+    </>
+  );
+};
+
+export const StickerItem = ({ title, children, id }) => {
+  return (
+    <StickerContainer>
+      <Stack
+        style={{ minHeight: 300 }}
+        justify="center"
+        align="center"
+        gap={SpaceSizes.md}
+      >
+        <Heading
+          as="a"
+          href={`/?path=/story/${id}`}
+          variant={HeadingVariants.secondary}
+          size={HeadingSizes.h5}
+        >
+          {title}
+        </Heading>
+        {children}
+      </Stack>
+    </StickerContainer>
+  );
+};
+
+export const StickerList = () => {
+  return (
+    <Grid cols={3} gap={SpaceSizes.lg}>
+      <StickerItem title="Accordion" id="components-accordion">
```

Comment:
We will need to update `id` properties to contain the whole story name. Good news is that except 4 components we can just add `--playground` at the end of the id (e.g. `components-accordion--playground`).

For Label, CloseButton, Icon, and Pagination you will need to check names in Storybook.